runtime/src/main/java/org/corfudb/runtime/view/Layout.java

```diff
@@ -329,8 +329,8 @@ public void validateSegmentSeal(LayoutSegment layoutSegment,
             }
 
             @Override
-            public int getMinReplicationFactor(Layout layout) {
-                return 2;
+            public int getMinReplicationFactor(Layout layout, LayoutStripe stripe) {
+                return 1;
```

Comment:
Would it make somehow sense for the replication factor to be defined/changed in the layout file? so it is not actually hard coded, and default to this if not present?

Reply:
The minimum replication factor should not change that often for it to be configurable. We hit a deadlock condition if the minimum replication factor is 2 in case of Chain replication.
On the other hand, the minimum replication factor would not be constant for quorum replication. It would increase as the number of nodes in the stripe increase and decrease when the nodes are removed.

Reply:
Ok, I was only thinking it for the chain replication but if this leads to potential deadlock then it would make sense to leave it out of the scope of the layout. 

Just to be sure, stripes are updated as a server becomes unresponsive? I'm just trying to make sure that stripes are not accounting for log servers that are down.

Reply:
Yes, the stripes are updated. If a node is down it is removed from the stripe. Provided the number of nodes remaining in the stripe is greater than or equal to the `minReplicationFactor`.
If not, the system just hangs until the condition is satisfied.